lib/help.txt

```diff
@@ -18,6 +18,9 @@ Options:
            --mock-dir, -d  A path to the directory containing mocks (by default,
                            test/image/mocks relative to the plotly directory)
 
+     -remote-topojson, -t  Use topojson from cdn.plot.ly (by default,
```

Comment:
Apologies for the duplication, but could you copy/paste this into the readme? I should figure out how to just template that automatically or something.